src/test/java/software/amazon/smithy/lsp/document/DocumentParserTest.java

```diff
@@ -11,30 +11,37 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 import static software.amazon.smithy.lsp.document.DocumentTest.safeIndex;
 import static software.amazon.smithy.lsp.document.DocumentTest.safeString;
 import static software.amazon.smithy.lsp.document.DocumentTest.string;
 
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 import org.eclipse.lsp4j.Position;
 import org.eclipse.lsp4j.Range;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 import software.amazon.smithy.lsp.protocol.LspAdapter;
 import software.amazon.smithy.model.Model;
 import software.amazon.smithy.model.SourceLocation;
 import software.amazon.smithy.model.shapes.Shape;
 
 public class DocumentParserTest {
+
     @Test
     public void jumpsToLines() {
         String text = """
                 abc
                 def
                 ghi
-
-
+                
+                
```

Comment:
What happened here?

Reply:
Ohh no!! 😞  I think we just need to add a good formatter! I used default formatting in intellij LOL.

Reply:
Yea agree